feeds/views.py

```
@@ -1 +1,64 @@
+            return qs
+        return qs
+
+
+@method_decorator(cache_page(10), name='list')
+class ExploreView(viewsets.GenericViewSet, ListModelMixin):
+    permission_classes = [AllowAny]
+    serializer_class = FeedThreadSerializer
+    pagination_class = FeedPagination
+
+
+    def get_queryset(self):
```

Flake8 (7.2.0)

[error] too many blank lines (2)

(E303)